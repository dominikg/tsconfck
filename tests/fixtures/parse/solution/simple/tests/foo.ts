import { foo } from '../src/foo.js';
import * as assert from 'assert';

function test() {
	const actual = foo();
	const expected = 'foo';
	assert.strictEqual(actual, expected);
}
test();
