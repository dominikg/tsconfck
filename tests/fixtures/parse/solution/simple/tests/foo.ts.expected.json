{
  "extends": "../tsconfig.base",
  "compilerOptions": {
    "composite": true,
    "strict": false,
    "strictNullChecks": true
  },
  "compileOnSave": false
}
